### DISCUSSION TITLE
Can SQLite-Web open a db outside of /root/homeassistant?

### DISCUSSION THREAD

**Discussion by busywait** [hassio-addons/addon-sqlite-web#309] [Q&A] [answered]
Is this addon restricted to only opening files in /homeassistant?

I copied a DB out of an add-on container (Music Assistant's library.db) to /media/library.db and configured SQLite-Web to open it.

```
homeassistant $ ls -l /media/
total 20388
...
-rw-r--r--    1 root     root      20832256 Apr 22 16:48 library.db
...
drwxr-xr-x  264 root     root         12288 Apr 23 07:18 music
```

SQLite Web options: 
```
database: /media/library.db
```

When I try to restart the SQLite Web addon I see this message in the logs:
```
[11:05:41] FATAL: The database file '/media/library.db' is not found
```

If I move library.db to /root/homeassistant and configure SQLite Web with `database: /homeassistant/library.db` then I can open and browse the DB.

Is there any other "shared data" location that I could have moved the DB to to explore it?

**Comment by frenck** [comment, marked as answer]
This add-on cannot access media.

See also: <https://github.com/hassio-addons/addon-sqlite-web/blob/6cbae78f09285be009021c3af082bbd213ebce45/sqlite-web/config.yaml#L16-L18>

**Comment by busywait** [comment]
Aha - makes sense, thx. Now I understand a little about HA add-on configuration, and will copy DB files to /share next time if I want to take a quick look inside them :)
